src/StimulusBundle/src/AssetMapper/ControllersMapGenerator.php

```diff
@@ -72,6 +72,9 @@ private function loadCustomControllers(): array
             $name = str_replace(['_', '/'], ['-', '--'], $name);
 
             $asset = $this->assetMapper->getAssetFromSourcePath($file->getRealPath());
+            if (null === $packageControllerConfig) {
```

Comment:
Should be `if (!$asset)` right? There is no `$packageControllerConfig` variable here.